supplier_api_tester/supplier_api_tester/utils/reservation.py

```diff
@@ -47,7 +47,12 @@ def get_reservation_slot(api_url, api_key, product_id, timeslots: bool, version=
     return single_variant_item
 
 
-def get_payload_from_slot(slot: Union[DailyAvailability, Timeslot], variant_quantity=1, min_quantity=1):
+def get_payload_from_slot(slot: Union[DailyVariants, Timeslot], variant_quantity=1, min_quantity=1):
+    print()
+    print("payload for slot is called")
+    print()
+    print("slot", slot, type(slot))
+    print()
```

Comment:
![](https://media.giphy.com/media/eiGU1eOEoxIIYL4I42/giphy.gif)

Reply:
will remove and also squash it, so the trace of it is gone

Reply:
I will remove it and squash the commits, will not be seen again